src/joystick/windows/SDL_rawinputjoystick.c

```diff
@@ -1313,13 +1313,11 @@ RAWINPUT_JoystickRumble(SDL_Joystick *joystick, Uint16 low_frequency_rumble, Uin
     }
 #endif /* SDL_JOYSTICK_RAWINPUT_XINPUT */
 
-    if (!rumbled) {
 #if defined(SDL_JOYSTICK_RAWINPUT_WGI) || defined(SDL_JOYSTICK_RAWINPUT_XINPUT)
+    if (!rumbled) {
         return SDL_SetError("Controller isn't correlated yet, try hitting a button first");
-#else
-        return SDL_Unsupported();
-#endif
     }
+#endif
     return 0;
```

Comment:
I am unsure if this is the most desired imlpementation for this.

Following the documentation we return -1 if rumble isn't supported on this joystick, but we cannot check that since neither WGI nor XInput is present.

Moving the rumbled definition out of the preprocessor might be better.

Reply:
If neither XInput nor WGI are available, then returning SDL_Unsupported() is the right thing to do here.